conf/routes

```diff
@@ -34,7 +34,10 @@ POST   /v3/users/self/externalId   	    @org.sagebionetworks.bridge.play.control
 GET    /v3/users/self/unsubscribeEmail  @org.sagebionetworks.bridge.play.controllers.EmailController.unsubscribeFromEmail
 POST   /v3/users/self/unsubscribeEmail  @org.sagebionetworks.bridge.play.controllers.EmailController.unsubscribeFromEmail
 POST   /v3/users/self/dataSharing       @org.sagebionetworks.bridge.play.controllers.ConsentController.changeSharingScope
-          
+
+# User data
+POST   /v3/userdata/request  @org.sagebionetworks.bridge.play.controllers.UserDataDownloadController.requestUserData
```

Comment:
Because the result is a zip file of many different types, I'm wondering about treating it as a manipulation of the user:

/v3/users/self/dataExport

/v3/userdata *could* expand with reports and the like, that would be an argument to keep it separate. At the moment though it's just something you can do to your own account, you can export the data.